.cmake/mito_tests_mito_lib.cmake

```diff
@@ -14,4 +24,23 @@ mito_test_driver(mito.lib/segmented-container/main.cc)
 mito_test_driver(mito.lib/segmented-container-iterator/main.cc)
 mito_test_driver(mito.lib/shared-pointer/main.cc)
 mito_test_driver(mito.lib/simplices/main.cc)
-mito_test_driver(mito.lib/traits/main.cc)
\ No newline at end of file
+mito_test_driver(mito.lib/traits/main.cc)
+
+if(WITH_VTK)
+    add_test(NAME tests.mito.lib.io.pytest
```

Comment:
I would wrap these additional tests into a `mito_test_driver_pytest_check` cmake function, so that we can simply call something like:
`mito_test_driver_pytest_check(mito.lib/io/pointCloudToVtk.cc)`

Reply:
Done